src/components/text/text.api.json

```diff
@@ -12,16 +12,182 @@
     "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Text/Transformation.png?raw=true"
   ],
   "props": [
-    {"name": "color", "type": "string", "description": "Color of the text"},
+    {
+      "name": "color",
+      "type": "string",
+      "description": "Color of the text",
+      "default": "Black",
+      "required": "true",
+      "platform": "iOS"
+    },
     {"name": "center", "type": "boolean", "description": "Whether to center the text (using textAlign)"},
     {"name": "uppercase", "type": "boolean", "description": "Whether to change the text to uppercase"},
     {"name": "underline", "type": "boolean", "description": "Whether to add an underline"},
-    {"name": "highlightString", "type": "HighlightString | HighlightString[]", "description": "Substring to highlight. Can be a simple string or a HighlightStringProps object, or an array of the above"},
+    {
+      "name": "highlightString",
+      "type": "HighlightString | HighlightString[]",
+      "description": "Substring to highlight. Can be a simple string or a HighlightStringProps object, or an array of the above"
+    },
     {"name": "highlightStyle", "type": "TextStyle", "description": "Custom highlight style for highlight string"},
     {"name": "recorderTag", "type": "'mask' | 'unmask'", "description": "Recorder Tag"},
     {"name": "animated", "type": "boolean", "description": "Use Animated.Text as a container"}
   ],
-  "snippet": [
-    "<Text text30$1>Text goes here$2</Text>"
-  ]
+  "snippet": ["<Text text30$1>Text goes here$2</Text>"],
+  "docs": {
+    "header": {
+      "description": "A wrapper for Text component with extra functionality like modifiers support",
+      "content": [
+        "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Docs/thumbnail-horizontal.png?raw=true"
+      ]
+    },
+    "tabs": [
+      { 
+        "title": "Overview",
+        "sections": [
+          {
+            "type": "table",
+            "title": "Table",
+            "description": "Examples of using the Text component",
+            "name": "Table Name",
+            "columns": ["Item", "Content[0]", "Content[1]"],
+            "content": [
+              {
+                "title": "Item #1",
+                "description": "Item #1 Subtitle",
+                "content": [
+                  "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Docs/thumbnail-horizontal.png?raw=true",
```

Comment:
Following our discussion this morning 
I suggest to at least have each item in the content array as an object, something like this 
```
"content": [
  {value: "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Docs/thumbnail-horizontal.png?raw=true"},
  {value: "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Docs/thumbnail-horizontal.png?raw=true",}
]

```

This will allow it to easily adapt to changes if we'll need more info (like type, or anything else) without changing the structure

Reply:
done